http-native/src/main/java/org/ballerinalang/net/http/mock/nonlistening/NonListeningRegister.java

```diff
@@ -18,38 +18,10 @@
 
 package org.ballerinalang.net.http.mock.nonlistening;
 
-import org.ballerinalang.jvm.api.values.BObject;
-import org.ballerinalang.jvm.scheduling.Scheduler;
-import org.ballerinalang.jvm.types.AttachedFunction;
-import org.ballerinalang.jvm.types.BType;
-import org.ballerinalang.net.http.HTTPServicesRegistry;
-import org.ballerinalang.net.http.HttpConstants;
-import org.ballerinalang.net.http.websocket.WebSocketConstants;
-import org.ballerinalang.net.http.websocket.server.WebSocketServerService;
-import org.ballerinalang.net.http.websocket.server.WebSocketServicesRegistry;
-
 /**
  * Register a service to the mock listener.
  *
  * @since 0.966
  */
 public class NonListeningRegister extends org.ballerinalang.net.http.serviceendpoint.Register {
```

Comment:
@chamil321 this class is not getting used, is this a mistake ?

Reply:
Yes, we don't need mock listeners anymore. Need to remove them.